bin/compile

```diff
@@ -12,6 +12,13 @@ if [ ! -f "$ENV_DIR/MAXMIND_DB_NAME" ]; then
   exit 1
 fi
 
+if [ ! -f "$ENV_DIR/MAXMIND_KEY" ]; then
+  echo "-----> You need to set the environment variable MAXMIND_KEY with your licence key from Maxmind."
```

Comment:
I suggest to include a link to the readme of this repository for help (`Check https://github.com/temedica/heroku-buildpack-maxmind-geolite2/blob/master/README.md for help.`).

And nitpick: MaxMind should be camelized. :)